crates/swc_ecma_minifier/tests/mangle/reuse/3/output.js

```diff
@@ -1267,9 +1267,9 @@ class aa extends Array {
         };
     }
 }
-class ba {
+class ab {
```

Comment:
Is this intentional?
Just curious, as the previous impl may be wrong.

Reply:
Yes.
It generate 
a, b, c ... z, A, B, C ... Z, $, _, 0,1, ... 9,
and
aa, ab, ac ...

In previous implementations, it seems that numbers were never used at the end.

Reply:
Oh, numeric endings are possible in previous implementation.
Now it is easier to implement `encode` and `decode` after changing the rules.